awesome_dashboard/static/src/dashboard/dashboard_statistics.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { reactive } from "@odoo/owl";
+
+const statisticsService = {
+    dependencies: ["rpc"],
+    start(env, {rpc}) { 
+        const statistics = reactive({ ready: false});
+
+        async function loadStatistics() {
+            const updates = await rpc("/awesome_dashboard/statistics");
+            Object.assign(statistics, updates, {ready: true});
+        }
+
+        setInterval(loadStatistics, 10 * 1000);
```

Comment:
```suggestion
        setInterval(loadStatistics, 10 * 60 * 1000);
```
The exercice asks for 10 minutes